src/saving_tools.jl

```diff
@@ -190,16 +192,18 @@ Dict{Symbol,Int64} with 2 entries:
   :y => 4
   :x => 3
 
-julia> tag!(d)
+julia> tag!(d, commit_message=true)
```

Comment:
```suggestion
julia> tag!(d; commit_message=true)
```
it is better syntax to separate keywords by `;`.